src/benchmarklib/tpch/tpch_table_generator.cpp

```diff
@@ -326,8 +330,20 @@ AbstractTableGenerator::IndexesByTable TPCHTableGenerator::_indexes_by_table() c
 }
 
 AbstractTableGenerator::SortOrderByTable TPCHTableGenerator::_sort_order_by_table() const {
-  // Allowed as per TPC-H Specification, paragraph 1.5.2
-  return {{"lineitem", "l_shipdate"}, {"orders", "o_orderdate"}};
+  if (_clustering_configuration == ClusteringConfiguration::Pruning) {
+    // This clustering improve the pruning of chunks for the two largest tables in TPC-H, lineitem and orders. Both
+    // tables are frequently filtered by the sorted columns, which improves the pruning rate significantly.
```

Comment:
Do we discuss the connection of clustering and sorting anywhere?

Reply:
Not more than that for the parameter of `hyriseBenchmarkTPCH`:
```
("clustering", "Clustering of TPC-H data. The default of --clustering=None means the data is stored as generated "
                   "by the TPC-H data generator. With --clustering=\"Pruning\", the two largest tables 'lineitem' "
                   "and 'orders' are sorted by 'l_shipdate' and 'o_orderdate' for improved chunk pruning. Both are "
                   "legal TPC-H input data.", cxxopts::value<std::string>()->default_value("None")); // NOLINT
```

Do you think we need more?

Reply:
No, that's fine